doc/source/ray-core/scheduling/accelerators.rst

```diff
@@ -4,7 +4,7 @@
 Accelerator Support
 ===================
 
-Accelerators (e.g. GPUs) are critical for many machine learning applications.
+Accelerators like GPUs are critical for many machine learning apps.
```

Comment:
"machine learning workloads" would be the better term here than "machine learning apps" I think :)